### PR TITLE
Add AliorBank.pl.xml

### PR DESCRIPTION
**Pull request by pipboy96**

### CODE REVIEW

File: src/chrome/content/rules/AliorBank.pl.xml [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+<ruleset name="AliorBank.pl">
+	<target host="aliorbank.pl" />
+	<target host="www.aliorbank.pl" />
+	<target host="4x4.aliorbank.pl" />
+	<target host="accelerator.aliorbank.pl" />
+	<target host="analityka.aliorbank.pl" />
+	<target host="ares.aliorbank.pl" />
+	<target host="auth.aliorbank.pl" />
+	<target host="autoryzacja.aliorbank.pl" />
+	<target host="wnioski.bm.aliorbank.pl" />
```
Comment (J0WI) [inline src/chrome/content/rules/AliorBank.pl.xml]: `broker.aliorbank.pl`

Comment (pipboy96) [reply src/chrome/content/rules/AliorBank.pl.xml]: How did you get that domain? Amass didn't find it.

Comment (J0WI) [reply src/chrome/content/rules/AliorBank.pl.xml]: search engine

Comment (pipboy96) [reply src/chrome/content/rules/AliorBank.pl.xml]: Thanks! I usually just use `amass enum -d domain.com --active --brute`.

**Pull request merged**